### ISSUE TITLE
how to solve statuscode:413 Request Entity Too Large

### ISSUE THREAD

**Issue by Koro-s-ensei** [strapi/strapi#8157]
**Describe the bug**
I have created the custom route and custom function to upload the the videos. The process to upload the is working fine for the less than 1mb files. But whenever I tried to upload file more than 1mb I got the error of 413 Request Entity Too Large. I'm using base64 encode to upload the videos. I have created the middle.js and request.json as documentation https://strapi.io/documentation/3.0.0-beta.x/concepts/middlewares.html#core-middlewares but it didn't help me. 

**Steps to reproduce the behavior**

1. create custom route and function in controller 
2. upload the video less than 1mb it is uploaded successfully  
3. when try to upload file more than 1mb 
4. got the error of 413:  Request Entity Too Large
5. created middleware.js and request.json 
6. but got the same error 

**Expected behavior**
after creating middleware and increase the size of jsonLimit in request.json. File will be uploaded but it didn't.

**Code snippets of request.json **
`{
	"parser": {
    "enabled": true,
    "multipart": true,
    "formLimit": "15mb",
    "jsonLimit": "15mb",
	"includeUnparsed": true
  }
}`
**Code snippets of middleware.js **
`module.exports = {
  load: {
   after: ['parser'],
  },
};`

**System**

- Node.js version: 12.18.3
- NPM version: 6.14.6
- Strapi version: 3.1.6
- Database: MySql
- Operating system: Windows 10

**Comment by derrickmehaffy** [comment]
I believe in your case the confusion is with the documentation, you are using the v3-stable but the document you are referencing is our old deprecated docs for the beta. See the new documentation here: https://strapi.io/documentation/v3.x/concepts/middlewares.html#request-middlewares

We are aware of an SEO issue with searching on google and such that is linking people to the old docs and are working to fix that.

Can you test with the new documentation and see if you are able to resolve the issue there? (If you are using a proxy like Nginx, it may also be an issue with the default 1MB `client_max_body_size` too)

**Comment by Koro-s-ensei** [comment]
I read the both documentations and followed both to create middleware.js and request.json . And I'm working locally I'm not using any proxy. This the path of my config file. I think I followed every step mentioned in the documentation.
 https://github.com/Koro-s-ensei/demi-2/tree/master/config

**Comment by laurenskling** [comment]
(https://github.com/strapi/strapi/issues/8146#issuecomment-702905633) 

I had the same thing, where the docs telling me to add formLimit and jsonLimit wasn't helping my raw body POST. I dug into the https://github.com/dlau/koa-body#options and found there's also textLimit which was the one i needed 👏

I also had to add includeUnparsed to make sure my XML was getting onto the body.

Hope it helps!

**Comment by derrickmehaffy** [comment]
> request.json

@Koro-s-ensei the request.json won't do anything for you on the stable release. It's an old config file that isn't called anymore. Can you try the solution mentioned by @laurenskling 

Also @laurenskling can you make a PR to expand the middleware docs to include that option?

**Comment by deepak19k96** [comment]
I have same issue. I have followed the @laurenskling  suggestion but its not working for me. 
please help me out this

**Comment by laurenskling** [comment]
https://strapi.io/documentation/v3.x/concepts/middlewares.html#core-middleware-configurations see where `parser` uses `koa-body` and try to figure out which settings you need.

**Comment by derrickmehaffy** [comment]
As no new information has been provided, marking as closed.

**Comment by krazylearner** [comment]
Getting same error 

`
<html>
<head><title>413 Request Entity Too Large</title></head>
<body>
<center><h1>413 Request Entity Too Large</h1></center>
<hr><center>nginx/1.20.0</center>
</body>
</html>
`

**Comment by vinzenzweber** [comment]
I am on 3.6.9 and using strapi-plugin-graphql. With GraphQL and Apollo Server, the issue actually is that the config is never passed down to the koa-bodyparser, which will fall back to '1mb'. [See this related issue](https://github.com/strapi/documentation/issues/498).
